e2e/tests/manageCases/createCase/FL401/fl401TheHome.spec.ts

```diff
@@ -0,0 +1,225 @@
+      subJourney: true,
+    });
+  });
+
+  test(`Complete the FL401 The Home Journey with the following options:
+        Not accessibility testing, 
+        Not Error Messaging (There isn't any), 
+        applicant has child
+        yes to all other booleans, 
+        'No' to has the applicant or respondent ever lived at the address, 
+        'Yes, Both of them' do they ever intend to live at the address`, async ({
+    page,
+  }): Promise<void> => {
+    await FL401TheHome.fl401TheHome({
+      page: page,
+      accessibilityTest: false,
+      applicantHasChildren: true,
+      fl401TheHomeYesNo: true,
+      fl401EverLivedAtAddress: "No",
+      fl401IntendToLiveAtAddress: "yesBothOfThem",
+      subJourney: true,
+    });
+  });
+
+  test(`Complete the FL401 The Home Journey with the following options:
+        Not accessibility testing, 
+        Not Error Messaging (There isn't any), 
+        applicant has child
+        yes to all other booleans, 
+        'Yes, the applicant' to has the applicant or respondent ever lived at the address`, async ({
+    page,
+  }): Promise<void> => {
+    await FL401TheHome.fl401TheHome({
+      page: page,
+      accessibilityTest: false,
+      applicantHasChildren: true,
+      fl401TheHomeYesNo: true,
+      fl401EverLivedAtAddress: "yesApplicant",
+      subJourney: true,
+    });
+  });
+
+  test(`Complete the FL401 The Home Journey with the following options:
+        Not accessibility testing, 
+        Not Error Messaging (There isn't any), 
+        applicant has child
+        yes to all other booleans, 
+        'Yes, the respondent' to has the applicant or respondent ever lived at the address`, async ({
+    page,
+  }): Promise<void> => {
+    await FL401TheHome.fl401TheHome({
+      page: page,
+      accessibilityTest: false,
+      applicantHasChildren: true,
+      fl401TheHomeYesNo: true,
+      fl401EverLivedAtAddress: "yesRespondent",
+      subJourney: true,
+    });
+  });
+
+  test(`Complete the FL401 The Home Journey with the following options:
+        Not accessibility testing, 
+        Not Error Messaging (There isn't any), 
+        applicant has child
+        yes to all other booleans, 
+        'Yes, both of them' to has the applicant or respondent ever lived at the address`, async ({
+    page,
+  }): Promise<void> => {
+    await FL401TheHome.fl401TheHome({
+      page: page,
+      accessibilityTest: false,
+      applicantHasChildren: true,
+      fl401TheHomeYesNo: true,
+      fl401EverLivedAtAddress: "yesBothOfThem",
+      subJourney: true,
+    });
+  });
+
+  test(`Complete the FL401 The Home Journey with the following options:
+        Not accessibility testing, 
+        Not Error Messaging (There isn't any), 
+        Applicant doesn't have children
+        No to all booleans, 
+        'No' to has the applicant or respondent ever lived at the address, 
+        'No' do they ever intend to live at the address, @crossbrowserManagecases`, async ({
```

Comment:
Does this need to be here

Reply:
they aren't boolean but I'll change above to 'no to all options'